internal/app/api/handler/login/login.go

```diff
@@ -50,3 +65,101 @@ func (h *LoginHandler) GetUser(w http.ResponseWriter, r *http.Request) {
 	respond.JSON(w, http.StatusAccepted, u)
 	return
 }
+func (h *LoginHandler) Register(w http.ResponseWriter, r *http.Request) {
+	username := r.FormValue("username")
```

Comment:
Why do you want to use form value for all of your API? Normal JSON body won't work or any reason?